src/core/MOM_forcing_type.F90

```diff
@@ -2496,6 +2489,9 @@ subroutine get_net_mass_forcing(fluxes, G, US, net_mass_src)
   if (associated(fluxes%lprec)) then ; do j=js,je ; do i=is,ie
     net_mass_src(i,j) = net_mass_src(i,j) + fluxes%lprec(i,j)
   enddo ; enddo ; endif
+  if (associated(fluxes%seaice_melt)) then ; do j=js,je ; do i=is,ie
```

Comment:
Moving up the contribution of `fluxes%seaice_melt` will change answers for people who are using the NUOPC coupler.  Adding the contributions from `fluxes%lprec` and `fluxes%seaice_melt` separately will change answers for people using the FMS coupler.  I am afraid that we are going to need a run-time logical flag to avoid changing somebody's answers.